open_event/static/js/admin/event/scheduling.js

```diff
@@ -0,0 +1,512 @@
+            $(event.target).addClass("drop-now");
+        },
+        ondragleave: function (event) {
+            $(event.target).removeClass("drop-now");
+        },
+        ondrop: function (event) {
+            var $sessionElement = $(event.relatedTarget);
+            var $trackDropZone = $(event.target);
+            $sessionElement.removeClass("unscheduled").addClass("scheduled");
+            $trackDropZone.removeClass("drop-active").removeClass("drop-now");
+
+            $sessionElement.css({
+                "-webkit-transform": "",
+                "transform": ""
+            }).removeData("x").removeData("y");
+
+            $sessionElement.appendTo($trackDropZone);
+            $sessionElement.css("top", $sessionElement.data("top") + "px");
+
+            var isColliding = sessionOverlapTest($trackDropZone, $sessionElement);
+            if (!isColliding) {
+                updateCounterBadge();
+                updateElementTime($sessionElement);
+                updateColor($sessionElement);
+            } else {
+                $sessionElement.appendTo($(".sessions-holder"));
+                $sessionElement.addClass("unscheduled").removeClass("scheduled").tooltip("hide").attr("data-original-title", "");
+                $sessionElement.css({
+                    "-webkit-transform": "",
+                    "transform": "",
+                    "background-color": ""
+                }).removeData("x").removeData("y");
+            }
+
+        },
+        ondropdeactivate: function (event) {
+            var $trackDropZone = $(event.target);
+            var $sessionElement = $(event.relatedTarget);
+            $trackDropZone.removeClass("drop-now").removeClass("drop-active");
+            if (!$sessionElement.hasClass("scheduled")) {
+                $sessionElement.css({
+                    "-webkit-transform": "",
+                    "transform": "",
+                    "background-color": ""
+                }).removeData("x").removeData("y").tooltip("hide").attr("data-original-title", "");
+            }
+        }
+    });
+}
+
+function minutesToPixels(minutes, forTop) {
+    minutes = Math.abs(minutes);
+    if (forTop) {
+        return ((minutes / time.unit.minutes) * time.unit.pixels) + time.unit.pixels;
+    } else {
+        return (minutes / time.unit.minutes) * time.unit.pixels;
+    }
+}
+
+function pixelsToMinutes(pixels, fromTop) {
+    pixels = Math.abs(pixels);
+    if (fromTop) {
+        return ((pixels - time.unit.pixels) / time.unit.pixels) * time.unit.minutes;
+    } else {
+        return (pixels / time.unit.pixels) * time.unit.minutes;
+    }
+}
+
+var days = [];
+var sessionsStore = [];
+var tracksStore = [];
+
+function processTrackSession(tracks, sessions, callback) {
+
+    var topTime = moment.utc({hour: time.start.hours, minute: time.start.minutes});
+
+    _.each(sessions, function (session) {
+
+        var startTime = moment.utc(session.start_time);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'start_time' is not in camel case.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2650806054/issues/source?bid=3233864&fileBranchId=3324174#l317)